### PR TITLE
Unit test for CloudExecutionStack

### PR DESCRIPTION
**Pull request by elharo**
@loosebazooka

### CODE REVIEW

**Comment by elharo** [comment]
ping

File: core-plugin/testSrc/com/google/gct/idea/debugger/CloudExecutionStackTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import javax.swing.Icon;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CloudExecutionStackTest {
+
+  @Mock
+  private Project project;
+
+  @Test
+  public void testGetTopFrame_null() {
```
Comment (patflynn) [inline core-plugin/testSrc/com/google/gct/idea/debugger/CloudExecutionStackTest.java]: testGetTopFrame_null() doesn't make it very clear what is supposed to be null or why.

File: core-plugin/testSrc/com/google/gct/idea/debugger/CloudExecutionStackTest.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    StackFrame frame2 = new StackFrame();
+    SourceLocation location2 = new SourceLocation();
+    location2.setLine(2);
+    frame2.setLocation(location2);
+
+    frames.add(frame1);
+    frames.add(frame2);
+
+    CloudExecutionStack stack = new CloudExecutionStack(project, "name", frames, null, null);
+    CloudStackFrame localFrame = (CloudStackFrame) stack.getTopFrame();
```
Comment (patflynn) [inline core-plugin/testSrc/com/google/gct/idea/debugger/CloudExecutionStackTest.java]: could we make stack.getTopFrame() covariant and return CloudStackFrame in its signature?

**Comment by patflynn** [comment]
LGTM once the test name suffix describes the test a little better.

File: core-plugin/testSrc/com/google/gct/idea/debugger/CloudExecutionStackTest.java (hunk 2 above)
Comment (elharo) [inline core-plugin/testSrc/com/google/gct/idea/debugger/CloudExecutionStackTest.java]: good idea; done

**Pull request merged**